pkg/model/elbv2/listener.go

```diff
@@ -69,6 +69,15 @@ const (
 	ProtocolTCP_UDP Protocol = "TCP_UDP"
 )
 
+type ProtocolVersion string
+
+const (
+	ProtocolVersionHTTP1 ProtocolVersion = "HTTP1"
+	ProtocolVersionHTTP2 ProtocolVersion = "HTTP2"
+	ProtocolVersionGRPC  ProtocolVersion = "GRPC"
+	ProtocolVersionNone  ProtocolVersion = ""
```

Comment:
i don't think we should have ProtocolVersionNone. let's make ProtocolVersion a `*ProtocolVersion` in the model since it's optional settings. (especially for tcp/tls case)